frontend/worker.ts

```diff
@@ -19,6 +19,15 @@ export default {
         return Response.redirect(new URL(`${APP_BASEPATH}/`, request.url), 302)
       }
 
+      // handle single fetch data loading and streaming format requests
+      // part of v3_singleFetch feature flag
+      if (url.pathname === '/tools.data') {
```

Comment:
This is the only route with such issue? Do we have an issue open in remix repo? Or maybe it was fixed in newer versions?